### ISSUE TITLE
Duplicated timeseries in CollectorRegistry

### ISSUE THREAD

**Issue by adampointer** [electrolux-oss/aws-cost-exporter#22]
Hi,

First, thanks for the great tool. This has been so useful for making our cost data more visible. 

I am getting a regular panic every other time the costs are queried from AWS.

```

  |   | 2024-11-21 02:12:25.547 | ValueError: Duplicated timeseries in CollectorRegistry: {'aws_daily_cost_by_service_by_account'} |  
  |   | 2024-11-21 02:12:25.547 | raise ValueError( |  
  |   | 2024-11-21 02:12:25.547 | File "/usr/local/lib/python3.10/site-packages/prometheus_client/registry.py", line 43, in register |  
  |   | 2024-11-21 02:12:25.547 | registry.register(self) |  
  |   | 2024-11-21 02:12:25.547 | File "/usr/local/lib/python3.10/site-packages/prometheus_client/metrics.py", line 143, in __init__ |  
  |   | 2024-11-21 02:12:25.547 | super().__init__( |  
  |   | 2024-11-21 02:12:25.546 | File "/usr/local/lib/python3.10/site-packages/prometheus_client/metrics.py", line 365, in __init__ |  
  |   | 2024-11-21 02:12:25.546 | self.aws_daily_cost_usd = Gauge(self.metric_name, "Daily cost of an AWS account in USD", self.labels) |  
  |   | 2024-11-21 02:12:25.546 | File "/app/exporter.py", line 40, in __init__ |  
  |   | 2024-11-21 02:12:25.546 | app_metrics = MetricExporter( |  
  |   | 2024-11-21 02:12:25.546 | File "//main.py", line 118, in main |  
  |   | 2024-11-21 02:12:25.546 | main(config) |  
  |   | 2024-11-21 02:12:25.546 | File "//main.py", line 138, in <module> |  
  |   | 2024-11-21 02:12:25.546 | Traceback (most recent call last):
```

I am using the default `polling_interval_seconds: 28800` setting. Do I need to tweak that setting?

**Comment by gluckzhang** [comment]
Hi @adampointer, the issue has been fixed by PR #23. I will wait a bit to release a new version because another open PR is almost done. Let's see if that one can be closed by today or tomorrow. I won't let you wait for too long :)

**Comment by gluckzhang** [comment]
Hi again, to not block you, I have released the fix in https://github.com/electrolux-oss/aws-cost-exporter/releases/tag/v1.0.7 Feel free to try it out and get back to me if there are other issues. I am going to close this one :)
